### PR TITLE
VZ-7460 Updating documentation for flexible LB shape

### PR DESCRIPTION
**Pull request by stevshap**
This PR updates the Verrazzano documentation to allow customization of oci-load-balancers using the `flexible` shape. Non-flex shapes are deprecated in May 2023

### CODE REVIEW

File: content/en/docs/customize/ingress.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ spec:
           contoller:
             service:
               annotations:
-                service.beta.kubernetes.io/oci-load-balancer-shape: 10Mbps
+                service.beta.kubernetes.io/oci-load-balancer-shape: flexible
+                service.beta.kubernetes.io/oci-load-balancer-shape-flex-max: "100"
+                 service.beta.kubernetes.io/oci-load-balancer-shape-flex-min: "100"
```
Comment (MelwynPaul) [inline content/en/docs/customize/ingress.md:33]: An extra space before the last new-line. 
Is that okay?  
We don't see that in the overrides section

File: content/en/docs/customize/ociLoadBalancerIPs.md [hunk 2]
```diff
@@ -21,15 +21,12 @@ This document describes how to use these annotations to customize the following
 ### Customize the load balancer shape  
 
 At installation, Verrazzano lets you customize the shape and size of the load balancers created.
-Oracle Cloud Infrastructure offers a flexible load balancer which uses Dynamic Shape:
-* 10 Mbps
-* 100 Mbps
-* 400 Mbps
-* 8,000 Mbps
+The shape of an Oracle Cloud Infrastructure load balancer specifies its maximum total bandwidth.
```
Comment (MelwynPaul) [inline content/en/docs/customize/ociLoadBalancerIPs.md:24]: Oracle Cloud Infrastructure > OCI 
(as the term is already defined)

**Review by MelwynPaul — COMMENTED** [review]
2 minor comments

**Comment by stevshap** [comment]
Resolved comments

**Review by MelwynPaul — APPROVED** [review]
Thank you.

**Pull request merged**